hexrd/ui/hexrd_config.py

```diff
@@ -874,7 +874,8 @@ def raw_masks_dict(self):
                 for det, mask in data:
                     if det == name:
                         final_mask = np.logical_and(final_mask, mask)
-            if self.threshold_mask_status:
+            if (self.threshold_mask_status and
+                    self.threshold_masks.get(name, None) is not None):
```

Comment:
```suggestion
                    self.threshold_masks.get(name) is not None):
```

If `self.threshold_masks` is a dictionary, then `get()` will return `None` by default if `name` is not present.

Reply:
Cleaned up :+1: